views/layouts/main.handlebars

```diff
@@ -12,19 +12,77 @@
 	<script src="https://code.jquery.com/jquery.js"></script>
   <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.3/umd/popper.min.js" integrity="sha384-ZMP7rVo3mIykV+2+9J3UJ46jBk0WLaUAdn689aCwoqbBJiSnjAK/l8WvCWPIPm49" crossorigin="anonymous"></script>
 	<script src="https://stackpath.bootstrapcdn.com/bootstrap/4.1.1/js/bootstrap.min.js" integrity="sha384-smHYKdLADwkXOn1EmN1qk/HfnUcbVRZyYmZ4qpPea6sjB/pTJ0euyQp0Mk8ck+5T" crossorigin="anonymous"></script>
-	
+	<style>
+        #map {
+        height: 200px;
+        width:900px;
+        }
+    </style>
 	<title>Project 2 Starter</title>
 </head>
 
 <body>
 	{{{ body }}}
-	<h2>My Google Maps Demo Featuring the Double Crown</h2>
+	<h2>Google Maps Demo TEST</h2>
     <!--The div element for the map -->
     <div id="map"></div>
+	
```

Comment:
I think we should write script in it's own .js file rather than including it in the handlebars file?

Reply:
I did that just to make it work. It didn't work when I had it in the server.js file.

Reply:
However it wouldn't live in the server.js file. It should live in its own .js file in the public/js folder. Might just need to push it for now and get it working and we can refactor it.